catalog/schematron/catalog-schematron-plugin/src/test/java/ddf/services/schematron/SchematronValidationServiceTest.java

```diff
@@ -121,11 +145,11 @@ private SchematronValidationService getService(boolean suppressWarnings, String
 
         ArrayList<String> schemaFiles = new ArrayList<>();
         for (String schematronFile : schematronFiles) {
+            String path = schematronFile;
             if (useClassLoader) {
```

Comment:
`if(useClassLoader)` is some code. It seems odd. Just an observation at this point.